app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.java

```diff
@@ -99,17 +105,16 @@ public static void open(@NonNull final Context context) {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(layout.activity_search);
+        ButterKnife.bind(this);
         Option.ofObj(savedInstanceState)
               .ifNone(this::addSearchFragment);
 
-        Toolbar toolbar = (Toolbar) findViewById(id.toolbar_search);
+        Toolbar toolbar = ButterKnife.findById(this, id.toolbar_search);
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
-        searchView = get((SearchView) findViewById(id.search_view));
-        searchView.setIconified(false);
-        closeButton = (ImageView) searchView
-                .findViewById(android.support.v7.appcompat.R.id.search_close_btn);
+        get(searchView).setIconified(false);
```

Comment:
On thing, if we would forget to add butterknife compiler, then there would no exception in the bind but the thing would be null.
Second thing, if I don't do this, I will have a warning about it being null.
I am opened for suggestions